Makefile

```diff
@@ -4,15 +4,20 @@
 MESSAGES = monocle/config.proto monocle/task_data.proto
 PINCLUDE = -I ./protos/ -I /usr/include
 
-codegen: codegen-python codegen-javascript codegen-stubs codegen-openapi
+codegen: codegen-python codegen-javascript codegen-stubs codegen-openapi codegen-haskell
 
 codegen-stubs:
 	mkdir -p srcgen/
-	(cd codegen; cabal run monocle-codegen ../protos/monocle/http.proto ../monocle/webapi.py ../srcgen/WebApi.res)
+	(cd codegen; cabal run monocle-codegen ../protos/monocle/http.proto ../src/Monocle/WebApi.hs ../monocle/webapi.py ../srcgen/WebApi.res)
+	~/.cabal/bin/ormolu -i ./src/Monocle/WebApi.hs
 	black ./monocle/webapi.py
 	./web/node_modules/.bin/bsc -format ./srcgen/WebApi.res > ./web/src/components/WebApi.res
 	rm -Rf srcgen/
 
+codegen-haskell:
+	sh -c 'for pb in $(MESSAGES); do ~/.cabal/bin/compile-proto-file --includeDir /usr/include --includeDir protos/ --proto $${pb} --out src/; done'
```

Comment:
could we output in haskell/src instead ?

Reply:
Sure.